components/Auction/index.js

```diff
@@ -108,78 +107,82 @@ const HandleAuctions = () => {
     e.target.src = '/assets/default_avatar.jpg';
   };
 
-  const auctionHandler = auctionsData.map((auction) => {
-    const { id, seller, quantity, highest_bid, bidders } = auction;
-    return (
-      <div className={`${styles.auctionContainer} ${id}`} key={id}>
-        <div className={styles.auctionSeller}>
-          <h2>Seller:</h2>
-          <img
-            className={styles.profilePhoto}
-            src={`${BASE_IMAGE_URL}/${seller}/img.png`}
-            onError={brokenImageHandler}
-          />
-        </div>
-        <div className={styles.auctionStats}>
-          <div className={styles.itemInfo}>
-            <h1>
-              {quantity} x {'  '}
-              <Image
-                className={styles.gemImage}
-                layout="fixed"
-                src="/assets/gem.png"
-                width={25}
-                height={25}
-              />
-            </h1>
-          </div>
-          <div className={styles.currentStatus}>
-            <h2>
-              Current Bid:
-              <div className={styles.currentBidPrice}>{highest_bid}</div>
-            </h2>
-          </div>
-        </div>
-        <div>
-          <form
-            className={styles.bidOptions}
-            onSubmit={(e) => handleNewBid(e, id)}
-          >
-            <input
-              className={styles.inputBid}
-              type="number"
-              min={parseInt(highest_bid) + 1}
-              required={true}
-              onBlur={({ target: { value } }) =>
-                validateBid(value, highest_bid)
-              }
-            />
-            <button type="submit" className={styles.bidBtn}>
-              Bid
-            </button>
-          </form>
-        </div>
-        <div className={styles.bidders}>
-          {bidders.map((bidder) => {
-            return (
-              <div
-                className={styles.biddersImg}
-                key={bidder}
-                data-columns={getColumns(bidders.length)}
-                title={bidder}
-              >
+  const auctionHandler =
+    auctionsData && auctionsData.length
+      ? auctionsData.map(({ id, seller, quantity, highest_bid, bidders }) => {
```

Comment:
Please create a sub-component which you can then use in the ternary operator...
Easier to read and understand the fallback, which in this case we have to scroll a lot to find 😅